tests/rptest/services/redpanda.py

```diff
@@ -286,7 +286,21 @@ def signal_redpanda(self, node, signal=signal.SIGKILL, idempotent=False):
 
         node.account.signal(pid, signal, allow_fail=False)
 
-    def start_node(self, node, override_cfg_params=None):
+    def lsof_node(self, node: ClusterNode):
```

Comment:
`lsof_node` should be in a separate commit. it's currently in `ducktape: Add custom timeout for stop/start redpanda node` with unrelated changes.